src/sentry/deletions/__init__.py

```diff
@@ -77,6 +78,8 @@ def load_defaults():
     default_manager.register(models.SavedSearchUserDefault, BulkModelDeletionTask)
     default_manager.register(models.Team, defaults.TeamDeletionTask)
     default_manager.register(models.UserReport, BulkModelDeletionTask)
+    default_manager.register(models.Widget, ModelDeletionTask)
```

Comment:
I changed this to `ModelDeletionTask` to have it follow the relations to `WidgetDataSource` in the organization deletion thing. If this is not the way to do it please let me know.

Reply:
@wedamija no it doesn't appear to be part of the usual pattern to be honest, but it's quite doable.